packages/pn-personafisica-webapp/src/navigation/SessionGuard.tsx

```diff
@@ -81,25 +83,35 @@ const SessionGuardRender = () => {
     message: isUnauthorizedUser ? messageUnauthorizedUser.message : hasErrorMessage.message,
   };
 
-  const renderIfInitialized = () =>
-    isUnauthorizedUser || hasTosPrivacyApiErrors || isClosedSession ? (
-      <SessionModal
-        open
-        title={goodbyeMessage.title}
-        message={goodbyeMessage.message}
-        handleClose={() => goToLoginPortal()}
-        initTimeout
-      />
-    ) : isAnonymousUser || DISABLE_INACTIVITY_HANDLER ? (
-      <Outlet />
-    ) : (
+  const renderIfInitialized = () => {
+    if (isUnauthorizedUser || hasTosPrivacyApiErrors || isClosedSession) {
+      return (
+        <SessionModal
+          open
+          title={goodbyeMessage.title}
+          message={goodbyeMessage.message}
+          handleClose={() => goToLoginPortal()}
+          initTimeout
+        />
+      );
+    } else if (isAnonymousUser) {
+      const aar = params.get(AppRouteParams.AAR);
```

Comment:
As before, in the pf we don't need the localStorage, but it's enough to send to the login portal with aar token as query param